### DISCUSSION TITLE
How can I customize SSHAddress in yaml file?

### DISCUSSION THREAD

**Discussion by pengbins** [lima-vm/lima#2959] [Q&A]
Is it possible to customize ssh.address in yaml file?  

```
unknown field \"address\"\n  152
```

**Comment by pengbins** [comment]
similar discussions

https://github.com/lima-vm/lima/discussions/2206

https://github.com/lima-vm/lima/discussions/1786

**Comment by AkihiroSuda** [comment]
The port number can be specified here:
https://github.com/lima-vm/lima/blob/09607e9698f7d3c0431795799e2bb2148ff4e5c1/templates/default.yaml#L144-L149


The listening IP address is currently hard-coded to 127.0.0.1. 
PR is welcome to support customization.

**Reply by afbjorklund** [reply]
We need to separate the local/listening address from the remote machine address, when adding external machines...

I used "address" for specifying the remote end (local is still 127.0.0.1), when doing the implementation: PR #2000

Maybe `localAddress` ?

Need to add (remote) "port"
